pkg/controller/provider/infoblox/access.go

```diff
@@ -71,6 +72,8 @@ func (this *access) NewRecord(fqdn string, rtype string, value string, zone prov
 			//Zone:     zone.Key(),
 			View: this.view,
 		}))
+	case dns.RS_AAAA:
+		log.Printf("warning: aaaa records not supported on infoblox yet")
```

Comment:
It is not sufficient to print a log line. There should be an explicit error and it should be reported in the DNSEntry status.

Reply:
I changed the interface to include errors and propagated it into the status.